Update user_management.md (#1348)
ezsystems/developer-documentation: docs/guide/user_management/user_management.md

You can set a rule that the password cannot be reused.
You set it for the User Content Type in the **User account (ezuser)** Field Type's settings.

> When this is set, the user cannot type in the same password as before, when it expires. It has to be changed to a new one. (Please note that this only checks the new password against the current one. A password that is older than this can be used again.)

Reviewer: ```suggestion
When this is set, the user cannot type in the same password when it expires.
It has to be changed to a new one.

This only checks the new password against the current one.
A password that has been used before can be used again.
```
^ if I understood correctly the "older password" thing
